pypeit/images/buildimage.py

```diff
@@ -182,11 +196,11 @@ def buildimage_fromlist(spectrograph, det, frame_par, file_list,
         finalImage = TraceImage.from_pypeitimage(pypeitImage)
     elif frame_par['frametype'] == 'align':
         finalImage = AlignImage.from_pypeitimage(pypeitImage)
-    elif frame_par['frametype'] in ['pixelflat', 'science', 'standard', 'illumflat']:
-        finalImage = pypeitImage
     else:
-        finalImage = None
-        embed(header=utils.embed_header())
+        finalImage = pypeitImage
+
+    # TODO: Can we move all this copying into the from_pypeitimage function,
```

Comment:
I think not, i.e. I probably tried and failed to do so..

Reply:
I'd like to leave this comment, just as a reminder for a latter PR.